cmd/thanos/tools_bucket.go

```diff
@@ -1420,3 +1453,90 @@ func registerBucketRetention(app extkingpin.AppClause, objStoreConfig *extflag.P
 		return nil
 	})
 }
+
+func registerBucketUploadSnapshotBlocks(app extkingpin.AppClause, objStoreConfig *extflag.PathOrContent) {
```

Comment:
Minor nit: this has nothing to do anymore with snapshots right?

Reply:
Ups! 😅 
Fixed